completions/screen

```diff
@@ -44,6 +44,9 @@ _screen()
             -*)
                 continue
                 ;;
+            /dev*)
+                continue
+                ;;
```

Comment:
The screen man page documents that the serial device must be the first argument, so I think it would be better to move the whole serial case to a block of its own, immediately after `_init_completion`, something like
```bash
if [[ $cword -eq 1 && $cur == /dev* ]]; then
```

Reply:
this is my first work with autocomplete!  whatever you think is best should be done, go ahead.  Please see my comment below about a bug with /dev/serial/by-id/... where the hyphen ("-") is causing interpretation to change to options.  Maybe your change would fix that?

Reply:
Nope, that didn't have anything to do with it, that's just what `ls` does:
```
/dev/ttyACM0

/dev/serial/by-id/:
usb-FIBOCOM_...
```